build.install4j

```diff
@@ -417,7 +588,7 @@
           </screen>
         </screens>
       </application>
-      <application name="" id="uninstaller" customizedId="" beanClass="com.install4j.runtime.beans.applications.UninstallerApplication" enabled="true" commentSet="false" comment="" actionElevationType="none" fileset="" customIcnsFile="" customIcoFile="" macEntitlementsFile="" automaticLauncherIntegration="false" launchMode="startupFirstWindow" launchInNewProcess="false" launchSchedule="updateSchedule" allLaunchers="true">
+      <application name="" id="uninstaller" customizedId="" beanClass="com.install4j.runtime.beans.applications.UninstallerApplication" enabled="true" commentSet="false" comment="" actionElevationType="elevated" fileset="" customIcnsFile="" customIcoFile="" macEntitlementsFile="" automaticLauncherIntegration="false" launchMode="startupFirstWindow" launchInNewProcess="false" launchSchedule="updateSchedule" allLaunchers="true">
```

Comment:
I ran the Windows x64 installer as a non-admin user.  I actually expected to get an elevation prompt but didn't see one.  Then I looked at `HKCR` to see if the `triplea` key was created, and I couldn't find it.  Here's the first registry-related error from the install log:

```
[INFO] com.install4j.runtime.beans.actions.registry.SetRegistryValueAction [ID 172]: Execute action
       Property value: URL:TripleA Protocol
       Property createKey: true
       Property valueName: 
       Property keyName: triplea
       Property registryRoot: HKEY_CLASSES_ROOT
       Property rollbackSupported: true
[ERROR] com.install4j.runtime.beans.actions.registry.SetRegistryValueAction [ID 172]: could not create key HKEY_CLASSES_ROOT triplea
        Execute action not successful after 1 ms
```

I will try rebuilding the installer with a root key of `HKCU\Software\Classes` instead of `HKCR` and see what happens.